astropy/io/votable/converters.py

```diff
@@ -1362,11 +1362,13 @@ def get_converter(field, config=None, pos=None):
     np.int64().dtype.num: "long",
     np.complex64().dtype.num: "floatComplex",
     np.complex128().dtype.num: "doubleComplex",
-    np.unicode_().dtype.num: "unicodeChar",
```

Comment:
We can just change this from `np.unicode_` to `np.str_` - they have been aliases since forever.